src/TDB2.cpp

```diff
@@ -355,6 +355,14 @@ void TF2::load_gc (Task& task)
   {
     Context::getContext ().tdb2.pending._tasks.push_back (task);
   }
+  // 2.6.0: Waiting status is deprecated. Convert to pending to upgrade status
+  // field value in the data files.
+  else if (status == "waiting")
```

Comment:
I like this, as it will modify all tasks on the next `gc`, rather than waiting until each individual task is modified.

Reply:
Indeed, eager upgrade procedure :slightly_smiling_face: